DuckDuckGo/AppDelegate.swift

```diff
@@ -45,6 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func applicationDidBecomeActive(_ application: UIApplication) {
         startMigration()
+        TrackerLoader.shared.updateTrackers()
```

Comment:
Cool, OK - that should be fine then - Alamofire handles that automatically?  

Well, it always seems fast for me, so I wasn't sure.  I'll watch it a bit closer and see what's going on, especially if there's a difference between the first and subsequent requests.